server/src/test/java/org/opensearch/node/InternalSettingsPreparerTests.java

```diff
@@ -114,6 +114,17 @@ public void testGarbageIsNotSwallowed() throws IOException {
         }
     }
 
+    public void testODFEConfigFileNotAllowed() throws IOException {
```

Comment:
A small thought, comparing to `ODFEConfigFile`, maybe using `ESConfigFile` is more general.

Reply:
Updated, good call.

Reply:
Thanks for the change. 😄

Reply:
^ It is a good idea, not kidding. 😉